simtools/data_model/validate_data.py

```diff
@@ -708,3 +710,25 @@ def _prepare_model_parameter(self):
                 if self.data_dict["unit"] is None
                 else gen.convert_string_to_list(self.data_dict["unit"])
             )
+
+    def _check_version_string(self, version):
+        """
+        Check that version string follows semantic versioning.
+
+        Parameters
+        ----------
+        version: str
+            version string
+
+        Raises
+        ------
+        ValueError
+            if version string does not follow semantic versioning
+
+        """
+        if version is None:
+            return
+        semver_regex = r"^\d+\.\d+\.\d+(-[0-9A-Za-z.-]+)?(\+[0-9A-Za-z.-]+)?$"
```

Comment:
Not perfectly familiar with regex. Does this here allow also number.number.number-(additional expression)?

Reply:
Yes - but see also #1115